### ISSUE TITLE
Events pagination does not have proper focus management

### ISSUE THREAD

**Issue by randimays** [department-of-veterans-affairs/va.gov-cms#14755]
## Description
When using the the pagination on events in a [VAMC listing](https://www.va.gov/minneapolis-health-care/events/?selectedOption=upcoming) or in [Outreach events](https://www.va.gov/outreach-and-events/events/), the screen should load with the top of the results in view so the veteran can find their place at the top of the list on the next or previous page. Focus should also be at the "Showing X - XX of XX results" so folks using assistive technology can easily orient themselves to the new list of results.

Currently, the behavior scrolls to the bottom of the page and the pagination numbers are at the top of the view.

The pagination & focus behavior should match [Find a Form](https://www.va.gov/find-forms/?q=disability) paging.

### User story
AS A Veteran
I WANT the page to scroll to the top when I use the pagination
SO THAT I can easily find my place in the list on the next page

Slack thread for context: https://dsva.slack.com/archives/C52CL1PKQ/p1691683090666189

## Acceptance criteria
- [x] When the user activates pagination links, the page should load with the top of the results list in view (as Find a Form currently does)
- [x] Focus should go to "Showing XX - XX of XX results" when the user activates pagination links (as Find a Form currently does)
- [x] Accessibility review

**Comment by jilladams** [comment]
Related ticket: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12089

**Comment by randimays** [comment]
Validated successfully in production. Closing!
